composer.json

```diff
@@ -25,7 +25,7 @@
         "jms/di-extra-bundle": "~1.4.0",
         "surfnet/stepup-middleware-client-bundle": "^1.3",
         "surfnet/stepup-saml-bundle": "^2.5",
-        "surfnet/stepup-bundle": "^1.5",
+        "surfnet/stepup-bundle": "^1.6",
```

Comment:
I often upgrade dependencies in a separate commit so it can easily be read back in the commit history. No need to `rebase -i` things, this is a personal preference.